fastlane/Fastfile

```diff
@@ -1,49 +1,105 @@
-# This file contains the fastlane.tools configuration
-# You can find the documentation at https://docs.fastlane.tools
-#
-# For a list of all available actions, check out
-#
-#     https://docs.fastlane.tools/actions
-#
-# For a list of all available plugins, check out
-#
-#     https://docs.fastlane.tools/plugins/available-plugins
-#
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
+require_relative './amplify_pods.rb'
 
 opt_out_usage
-
 default_platform(:ios)
 
-COCOAPODS_PUSH_WAIT_TIME = 600
+PLIST_KEY = "CFBundleShortVersionString"
 
-pods = [
-  {spec: "Amplify.podspec", sleep: true},
-  # AmplifyPlugins depends on AWSPluginsCore, but skipping the sleep here
-  # since there's one after CoreMLPredictionsPlugin
-  {spec: "AWSPluginsCore.podspec", sleep: false},
-  {spec: "CoreMLPredictionsPlugin.podspec", sleep: true},
-  {spec: "AWSPredictionsPlugin.podspec", sleep: false},
-  {spec: "AmplifyPlugins.podspec", sleep: false}
-]
+COCOAPODS_PUSH_WAIT_TIME = 600
```

Comment:
Is this still necessary? I recall we were going to try the new `--synchronous` flag on our pod pushes?

Reply:
I just opened a PR for that fastlane action, adding the synchronous option: https://github.com/fastlane/fastlane/pull/16698

Of course, we can just use a bash command until that gets merged

Reply:
update: I've removed the constant and `sleep` logic. Calling `pod trunk push` with `sh` will update to action once someone on the fastlane team merges my PR